packages/components/pie-webc-core/src/mixins/formControl/formControlMixin.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-classes-per-file */
+import { LitElement } from 'lit';
+import type { GenericConstructor } from '../types/GenericConstructor';
+
+/**
+ * Interface for FormControl behavior.
+ */
+declare class _FormControlInterface {
```

Comment:
Something for another PR, perhaps, but I've done a bit of reading and it seems like `declare class` is roughly the same as `interface`, so I wonder if you could use that here instead? It may also remove the need to disable the `max-classes-per-file` rule.

Reply:
fully agreed - updated to use an interface!